aptos-move/vm-genesis/src/lib.rs

```diff
@@ -190,11 +192,9 @@ fn create_and_initialize_main_accounts(
 ) {
     let aptos_root_auth_key = AuthenticationKey::ed25519(aptos_root_key);
 
-    let genesis_gas_schedule = &INITIAL_COST_SCHEDULE;
-    let instr_gas_costs = bcs::to_bytes(&genesis_gas_schedule.instruction_table)
-        .expect("Failure serializing genesis instr gas costs");
-    let native_gas_costs = bcs::to_bytes(&genesis_gas_schedule.native_table)
-        .expect("Failure serializing genesis native gas costs");
+    let genesis_gas_params = AptosGasParameters::initial();
```

Comment:
Are you going to make this configurable in the genesis blob in a followup PR?

Reply:
I assume it's already configurable in the aptos-gas crate? The numbers you see are for genesis.

Reply:
Hmm that's a fair point. I was thinking of having different gas schedules for different testnets, but probably not a use case in demand yet as we want testnet to be as realistic as possible anyway so it makes sense to keep them the same for now.

Reply:
In addition to what you mentioned, I also think that tests should have their own gas schedule defined. Otherwise they can be quite sensitive to gas schedule changes and it would be extremely hard to debug.